### ISSUE TITLE
Guild

### ISSUE THREAD

**Issue by firelook** [AshamaneProject/AshamaneCore#186]
**Description:**

As a guild master you cna nnot change any roles and resposanbilities to guild level characters to grant them gold, reparation,... (all items linked to grade like initiate)

**Current behaviour:**
box can not be checked and saved, gold can not be modified

**Expected behaviour:**

to be allowed to change items and saved

found with latest version in github as of 04/10/2019

**Comment by MttAI-dev** [comment]
that bug u are talking was on 8.0.1 0n 8.2 it works perfect. maybe u have some custom base update? there are people who use foreign files, modify and share them as part of their work.

**Comment by firelook** [comment]
not at all, and also the server would not lauch if some sql files were missing ?
Also seems more a code problem raher a sql problem (can you point me to the file to allow me to check please ?)
perhaps it is the ocassion to merge all SQLs ?
